addons/report/controllers/main.py

```diff
@@ -12,6 +8,10 @@
 from werkzeug.datastructures import Headers
 from reportlab.graphics.barcode import createBarcodeDrawing
 
+from odoo.http import Controller, route, request
+from odoo.addons.web.controllers.main import _serialize_exception, content_disposition
+from odoo.tools import html_escape
+
```

Comment:
Yo man please its really silly please arrange it... A for Apple or B for Ball :laughing: